sriov_network_operator/sriov_network_operator_ci_start.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash -x
+
+source ./common/common_functions.sh
+
+export RECLONE=${RECLONE:-true}
+export WORKSPACE=${WORKSPACE:-/tmp/k8s_$$}
+export LOGDIR=$WORKSPACE/logs
+export ARTIFACTS=$WORKSPACE/artifacts
+export TIMEOUT=${TIMEOUT:-300}
+export POLL_INTERVAL=${POLL_INTERVAL:-10}
+
+export SRIOV_NETWORK_OPERATOR_REPO=${ANTREA_CNI_REPO:-https://github.com/k8snetworkplumbingwg/sriov-network-operator.git}
+export SRIOV_NETWORK_OPERATOR_BRANCH=${ANTREA_CNI_BRANCH:-''}
+export SRIOV_NETWORK_OPERATOR_PR=${ANTREA_CNI_PR:-'97'}
+export SRIOV_NETWORK_OPERATOR_HARBOR_IMAGE=''
+
+export SRIOV_CNI_IMAGE='nfvpe/sriov-cni:v2.6'
+export SRIOV_INFINIBAND_CNI_IMAGE='mellanox/ib-sriov-cni:faa9e36'
+export SRIOV_DEVICE_PLUGIN_IMAGE='quay.io/openshift/origin-sriov-network-device-plugin:4.8'
+export SRIOV_NETWORK_CONFIG_DAEMON_IMAGE='mellanox/sriov-operator-daemon:ci'
+export SRIOV_NETWORK_OPERATOR_IMAGE='mellanox/sriov-operator:ci'
+
+export GOPATH=${WORKSPACE}
+export PATH=/usr/local/go/bin/:$GOPATH/src/k8s.io/kubernetes/third_party/etcd:$PATH
+
+export CNI_BIN_DIR=${CNI_BIN_DIR:-/opt/cni/bin/}
+export CNI_CONF_DIR=${CNI_CONF_DIR:-/etc/cni/net.d/}
+export KUBECONFIG='/root/.kube/config'
+
+#TODO add autodiscovering
+export SRIOV_INTERFACE=${SRIOV_INTERFACE:-auto_detect}
+export VFS_NUM=${VFS_NUM:-4}
+
+function download_and_build {
+    status=0
+    if [ "$RECLONE" != true ] ; then
+        return $status
+    fi
+
+    export DOCKERFILE=Dockerfile
+    export IMAGE_TAG="$SRIOV_NETWORK_OPERATOR_IMAGE"
+    build_github_project "sriov-network-operator" "make image"
+    let status=status+$?
+    if [ "$status" != 0 ]; then
+        echo "ERROR: Failed to build network operator iamge: $SRIOV_NETWORK_OPERATOR_IMAGE!"
+        return $status
+    fi 
+
+    upload_image_to_kind "$SRIOV_NETWORK_OPERATOR_IMAGE"
+    let status=status+$?
+    if [[ "$status" != "0" ]];then
+        echo "ERROR: Failed to upload iamge $SRIOV_NETWORK_OPERATOR_IMAGE! to kind!"
+        return $status
+    fi
+
+    export DOCKERFILE=Dockerfile.sriov-network-config-daemon
+    export IMAGE_TAG="$SRIOV_NETWORK_CONFIG_DAEMON_IMAGE"
+    build_github_project "sriov-network-operator" "make image"
+    let status=status+$?
+    if [ "$status" != 0 ]; then
+        echo "ERROR: Failed to build network operator config-daemon iamge: $SRIOV_NETWORK_CONFIG_DAEMON_IMAGE!"
+        return $status
+    fi
+
+    upload_image_to_kind "$SRIOV_NETWORK_CONFIG_DAEMON_IMAGE"
+    let status=status+$?
+    if [[ "$status" != "0" ]];then
+        echo "ERROR: Failed to upload iamge $SRIOV_NETWORK_CONFIG_DAEMON_IMAGE! to kind!"
+        return $status
+    fi
+    
+    return $status
+}
+
+function deploy_sriov_network_operator {
+    local status=0
+
+    pushd "$WORKSPACE/sriov-network-operator"
+
+    label_node
+
+    make deploy-setup-k8s
```

Comment:
you didn't specify the operator `SRIOV_NETWORK_OPERATOR_IMAGE ` and daemon `SRIOV_NETWORK_CONFIG_DAEMON_IMAGE ` images and the cluster type `CLUSTER_TYPE`

sriov operator uses latest master if they are not speicfied and cluster type to openshift

Reply:
see [Makefile](https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/Makefile#L195-L200) for basic env variables, and see [vars](https://github.com/Mellanox/kubernetes-ci/pull/105/files?file-filters%5B%5D=.sh#diff-1da590bb51c16903726785c833421c5daa35025cdc36c923aca64459aae22f1cR17-R21) for the image variables

Reply:
I see you set the env vars for images, what about the cluster type?

Reply:
it is set in the Makefile of the sriov-network operator